wiki/Installing-drivers-and-fix-them.md

```diff
@@ -0,0 +1,77 @@
+
+## Method 1: Check Windows Update
+You can usually find drivers and drivers update in Windows Update, instead of manually downloading the file.
+ 1. Open the Settings app using the Start menu.
+ 2. Go to Updates and security.
+ 3. Click on “Check updates”.
+ 4. Windows Update might say that you are up-to-date, even if optional updates are available. In this case, you will see an “Optional Updates” button. Clicking on it will show a list of updates (all of them are typically driver updates). Select all of them and click on “Download and install”.
+
+## Method 2: Download using the manufacturer's website.
+It's highly recommended to download or update the drivers, through the manufacturer's website, as it's guaranteed that they will work correctly. This is especially a good idea for the GPU (Graphics) drivers.
+All you need to do is:
+
+ 1. On the search engine of your choice (Google, Bing, etc…), search for "{device manufacturer} drivers".
+ 2. Next, enter the ID of the device and follow the instructions on the page.
+ 3. When you select the product, you have two options:
+	- Download the driver downloader (recommended)
+	- And download the driver on specific and run executable
+
+## Method 3: Download using the OEM's website.
+If you want to download the drivers for your GPU (Graphics Card), and the manufacturer's website doesn't provide the drivers, you can download them from the OEM's website. Only do this if you have to.
+Now follow the same instructions as Method 2, but with the OEM's website.
+
+ 1. On the search engine of your choice (Google, Bing, etc…), search for "{the OEM's name} drivers".
+ 2. Search for the model number of the piece of hardware you want to upgrade/install drivers for.
+ 3. When you select the product, you have two options:
+	- It's recommended to use the driver downloader if it's provided, as it makes the installation a lot easier.
+	- Download the driver on specific and run the executable
+- See [Reinstalling GPU drivers from the OEM's website](https://msft.chat/wiki/reinstalling-gpu-drivers.html#method-3-oem-website) for more information.
+## Troubleshooting 🔧
+
+1. Enter Safe Mode:
+
+Safe mode is a diagnostic tool, which loads Windows with only the essential drivers. This allows you to access Windows even if the problematic driver is causing issues during normal boot.
+
+ - Windows 10/11: Repeatedly press and hold the power button while booting to trigger a forced restart. After 2–3 restarts, you'll see the “Advanced options” screen. Navigate to “Troubleshoot” > “Advanced options” > “Startup Settings” > “Restart now”. Upon restart, choose “Enable Safe Mode with Networking” (if you need internet) or simply “Safe Mode”.
```

Comment:
Maybe mention holding shift while pressing the restart button?

Reply:
how are u gonna do that if ur
gpu drivers are no worky worky and u cant see smh

Reply:
> Maybe mention holding shift while pressing the restart button?

In some cases that doesn't work (happened to me).

Reply:
just suggesting to mention that, since it's less obnoxious than having to restart 3 times

Reply:
@julsehn It is still worth mentioning; just because it doesn't always work for you, doesn't mean it's not a solution worth trying.

@sthivaios I didn't interpret this section as being troubleshooting steps for when GPU drivers fail specfically? If it's another driver that isn't impeding your ability to Shift+Restart into WinRE, you'd still be able to do the steps. This section also appears to mention the button in Settings to restart into WinRE anyway, so if the article mentions that method, might as well also mention this one.

Reply:
fair enough @FloatingMilkshake